pyEpiabm/pyEpiabm/core/person.py

```diff
@@ -198,3 +198,17 @@ def is_place_closed(self, closure_place_type):
                 if place_type.value in closure_place_type:
                     return True
         return False
+
+    def remove_person(self):
+        """ Method to remove Person object from population.
```

Comment:
You should explain that this functionality is for travellers - for example dead individuals still count as part of the population

Reply:
Added a line saying this.